posthog/api/test/test_plugin.py

```diff
@@ -191,7 +191,7 @@ def test_update_plugin_auth_to_globally_managed(self, mock_get, mock_reload):
                 self.organization.plugins_access_level = level
                 self.organization.save()
                 response = self.client.patch(
-                    f"/api/organizations/@current/plugins/{install_response.json()['id']}/", {"is_global": False}
+                    f"/api/organizations/@current/plugins/{install_response.json()['id']}/", {"is_global": is_global}
```

Comment:
This is a change in the test behaviour but looking at it I suspect the previous implementation could be a bug.